src/OpenTracing/GlobalTracer.php

```diff
@@ -30,7 +30,7 @@ public static function setGlobalTracer(Tracer $tracer)
      *
      * @return Tracer
      */
-    public static function getGlobalTracer()
+    public static function get()
     {
         if (self::$globalTracerInstance === null) {
             self::$globalTracerInstance = NoopTracer::create();
```

Comment:
may also wanna rename this to just `self::$instance`, but not so important as it's private